### ISSUE TITLE
dip(xx) is not displaying well under different density

### ISSUE THREAD

**Issue by Anthonyeef** [Kotlin/anko#221]
We have a project under developing and publishing for 5 years, and this year we decided to take a bite of  Kotlin. And of course, we tried to use Anko for layout construction.

Later I found something strange. Some components in our project is written with Java and xml, some similar components I tried to convert to Kotlin with Anko. No xml for layout anymore and it looks pretty, while later I figured out that some component display differently within different density devices(compare to what it looks like before using Java and xml). In another word, when I use `dip(15)` with anko for TextView layout width, it behaves differently between different device density.

More details could be provided, if it's necessary.

**Comment by fboldog** [comment]
Could you provide a simplified example (xml vs anko)?

**Comment by Anthonyeef** [comment]
@fboldog  sure. The project I mentioned above is quite business thus I can't directly post its code and screenshot here, but I am willing to create a project to demonstrate what the issue looks like.

Thx for the reply, btw ;-)

**Comment by Anthonyeef** [comment]
I finally figured out what's going wrong there in our project. It's our fault and I think Anko do it in the right way.

For some reason we decided to use dp as the font unit in our app. We are making a digital reader application and our designer somehow wish our user don't try to change font scale via system settings( so we use dp unit which won't change font size once user try to change it via system setting). And when we use Anko for TextView in our app, anko will apply default sp unit for TextView, which cause some different display results.

It should not be considerd as Anko's bug, it's more like my misunderstood of something. And for that reason I mentioned above, I think I have to continue use dp as the unit of our app( and I just found that it's simple with Kotlin's extension method)

I will close this issue. Thanks for taking a look at this issue( and figured out it's actually not an issue at all...
